test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -6071,10 +6086,18 @@ def submit(self, obj, script=None, extend=None, submit_dir=None,
                 runcmd = [
                     'PBS_CONF_FILE=' + self.client_pbs_conf_file] + runcmd
                 as_script = True
-
+            if len(env_set) > 0:
+                user = PbsUser.get_user(TEST_USER)
```

Comment:
TEST_USER is the default user.  Shouldn't you use the user from the job/resv object?